core/src/scene/styleContext.cpp

```diff
@@ -25,20 +24,8 @@ static const std::string key_zoom("$zoom");
 StyleContext::StyleContext() {
     m_ctx = duk_create_heap_default();
 
-    // add empty feature_object
-    duk_push_object(m_ctx);
-
-    // assign instance to feature_object
-    duk_push_pointer(m_ctx, this);
-    if (!duk_put_prop_string(m_ctx, -2, DATA_ID)) {
-        LOGE("Ctx not assigned");
-    }
-
-    // put object in global scope
-    if (!duk_put_global_string(m_ctx, "feature")) {
-        LOGE("Feature not assigned");
-    }
-
+    //// Create global geometry constants
+    // TODO make immutable
```

Comment:
Yeah, this can be done later - there is also Object.freeze() which sounds related